specs/CustomScrollbar.md

```diff
@@ -0,0 +1,88 @@
+Custom Scrollbar 
+===
+
+# Background
+Developers would like to be able to set the scrollbar to Windows overlay style and match the Windows theme, but browser flags keep changing and are not reliable. Thus we want to provide an extendable API that's allow developers to define a scrollbar style in their app.
+
+# Examples
+## WinRT and .NET   
+```c#
+/// Create WebView Environment with option
+async void CreateEnvironmentWithOption()
+{
+    CoreWebView2EnvironmentOptions options = new CoreWebView2EnvironmentOptions();
+    options.CustomScrollbarStyle = COREWEBVIEW2_SCROLLBAR_WIN_FLUENT_OVERLAY_STYLE;
+    CoreWebView2Environment environment = await CoreWebView2Environment.CreateAsync(options: options);
+    webview.EnsureCoreWebView2Async(environment);
+}
+```
+
+## Win32 C++
+```cpp
+void AppWindow::InitializeWebView()
+{
+    auto options = Microsoft::WRL::Make<CoreWebView2EnvironmentOptions>();
+    COREWEBVIEW2_SCROLLBAR_STYLE style = COREWEBVIEW2_SCROLLBAR_WIN_FLUENT_OVERLAY_STYLE;
+    CHECK_FAILURE(options->SetCustomScrollbarStyle(styles));
+
+    // ... other option properties
+
+    // ... CreateCoreWebView2EnvironmentWithOptions
+}
+```
+
+# API Notes
+
+See [API Details](#api-details) section below for API reference.
+
+# API Details
+## Win32 C++
+
+```IDL
+interface ICoreWebView2EnvironmentOptions7;
+
+[v1_enum]
+typedef enum COREWEBVIEW2_SCROLLBAR_STYLE {
+  /// Browser default style
+  COREWEBVIEW2_SCROLLBAR_BROWSER_DEFAULT,
+  /// Window Style fluent overlay scroll bar
+  COREWEBVIEW2_SCROLLBAR_WIN_FLUENT_OVERLAY_STYLE
```

Comment:
A few notes:
* Don't abbreviate Windows to Win.
* Need the full enum name as a prefix in the enum value names (this applies to both enum value names)
* Having Style at the end seems like duplicated naming.


```suggestion
  COREWEBVIEW2_SCROLLBAR_STYLE_WINDOWS_FLUENT_OVERLAY
```